OpenSim/Auxiliary/auxiliaryTestFunctions.h

```diff
@@ -77,18 +77,18 @@ inline void ASSERT(bool cond,
  * specified tolerances. If RMS error for any column is outside the
  * tolerance, throw an Exception.
  */
-void CHECK_STORAGE_AGAINST_STANDARD(OpenSim::Storage& result, 
-                                    OpenSim::Storage& standard, 
-                                    OpenSim::Array<double> tolerances, 
-                                    std::string testFile, 
-                                    int testFileLine, 
-                                    std::string errorMessage)
+void CHECK_STORAGE_AGAINST_STANDARD(const OpenSim::Storage& result, 
+                                    const OpenSim::Storage& standard, 
+                                    const std::vector<double>& tolerances, 
+                                    const std::string testFile, 
```

Comment:
Ah, yes. Thanks, I got a little distracted by the `const` and forgot the references there. This also made me look at the `int` line, and apparently there's some thought that passing primitives by reference may not speed things up (one of many sources [here](http://stackoverflow.com/questions/3009543/passing-integers-as-constant-references-versus-copying))